build/media-autobuild_suite.ini

```diff
@@ -41,12 +41,12 @@ jo=2
 dssim=2
 avs2=2
 CC=2
-cores=4
+cores=2
```

Comment:
Github runners only has 2 cores, for local builds one has to change this for now, we can maintain 2 separate configs if necessary.